src/integration-tests/src/compute.rs

```diff
@@ -101,27 +109,31 @@ pub async fn machine_types() -> Result<()> {
     Ok(())
 }
 
-pub async fn images() -> Result<()> {
+async fn latest_image(client: &Images, project_id: &str, prefix: &str) -> Result<String> {
     use compute::model::image::Architecture;
 
-    let client = Images::builder().with_tracing().build().await?;
-
-    tracing::info!("Testing Images::list()");
     let mut latest = None;
-    let mut items = client.list().set_project("cos-cloud").by_item();
+    let mut items = client.list().set_project(project_id).by_item();
     while let Some(item) = items.next().await.transpose()? {
         tracing::info!("item = {item:?}");
         if item.architecture != Some(Architecture::X8664)
             || item
                 .family
-                .as_ref()
-                .is_some_and(|v| v.strip_prefix("cos-").is_some())
+                .as_deref()
+                .is_some_and(|v| v.strip_prefix(prefix).is_none())
         {
             continue;
         }
         latest = match &latest {
             None => Some(item),
-            Some(i) if item.family > i.family => Some(item),
+            Some(i)
+                if item
+                    .family
+                    .as_deref()
+                    .is_some_and(|f| f > i.family.as_deref().unwrap_or_default()) =>
```

Comment:
nit: looking more closely, this is probably not always correct. e.g. `foo-9` > `foo-10`.

Nobody cares though.